pegasus/forms/hoc_signup_2020.rb

```diff
@@ -2,6 +2,39 @@
 
 #HOC Sign up form for 2020. 2017 was the last form with new logic
 class HocSignup2020 < HocSignup2017
+  def self.normalize(data)
+    Honeybadger.context({data: data})
```

Comment:
Interesting.  What does this do?

Reply:
This was a function that existed in the previous HoCSignup versions--it validates/cleans up the form content. I had to copy it over because we added an allowed value to the event types, which would be rejected if we tried to call the previous version.